src/BloomBrowserUI/bookEdit/BookPagesThumbnailList/BookPagesThumbnailList.htm

```diff
@@ -31,12 +31,30 @@
             jQuery('.gridItem').click(function () {
                 fireCSharpEvent("gridClick", $(this).attr('id'));
             });
+
+            setTimeout(loadNextThumbnail, 100);
         });
         function fireCSharpEvent(eventName, eventData) {
 
             var event = new MessageEvent(eventName, { 'view': window, 'bubbles': true, 'cancelable': true, 'data': eventData });
             document.dispatchEvent(event);
         }
+
+        function loadNextThumbnail() {
```

Comment:
Even with the restricted number of threads, it seems to work better if there is a delay. I believe I reduced the time out to 20 ms in that pull request.